sched/pthread/pthread_mutex.c

```diff
@@ -38,6 +38,12 @@
 #include "sched/sched.h"
 #include "pthread/pthread.h"
 
+/****************************************************************************
+ * Public Data
+ ****************************************************************************/
+
+spinlock_t mutex_spinlock = SP_UNLOCKED;
```

Comment:
```suggestion
spinlock_t g_mutex_spinlock = SP_UNLOCKED;
```

Reply:
any elegant way to remove this global variable?

Reply:
I have made g_mutex_spinlock as  static.